guides/source/active_record_migrations.md

```diff
@@ -612,10 +612,12 @@ definitions:
 * [`add_index`][]
 * [`add_reference`][]
 * [`add_timestamps`][]
+* [`change_column_comment`][] (must supply a `:from` and `:to` option)
```

Comment:
To use the reference-style link syntax (`[link text][]`), both `change_column_comment` and `change_table_comment` need their URLs defined elsewhere.  Let's add them to the block of URLs on lines 648-658.

Reply:
Thanks for the guidance @jonathanhefner . I'm happier with the changes now.